FABulous/fabric_generator/file_parser.py

```diff
@@ -579,45 +582,30 @@ def parseFile(filename: str, belPrefix: str = "", filetype: str = "") -> Bel:
         exit(-1)
 
     if filetype not in ("verilog", "vhdl"):
```

Comment:
You can add a `Literial` type at the function argument, which make the function more readable